abi-types-generator/src/converters/typescript/ethers-factory.ts

```diff
@@ -105,7 +105,7 @@ export class EthersFactory {
       parameters += ', ';
     }
 
-    if (abiItem.constant === true) {
+    if (abiItem.constant === true || abiItem.stateMutability === 'view' || abiItem.stateMutability === 'pure') {
```

Comment:
We use this logic twice now want to create a helper method (in common > helpers) which passes in an `AbiItem` and does this logic returning a boolean.. this can be shared on `web3-factory.ts` then and avoid 2 of the same code. `IsConstantOrStateMutabilityViewOrPure` or something along those lines if you have a better name for it.